.github/ISSUE_TEMPLATE/happiness-bug-report.yml

```diff
@@ -6,21 +6,21 @@ body:
     attributes:
       value: |
         <br>
-         
-        ### Thanks for contributing! 
-        
+
+        ### Thanks for contributing!
+
         Please write a clear title, then fill in the fields below and submit.
         There are only ***3 required fields***, but please provide as much additional information and context as you are able to.
-        
-        Avoid using image hosting services such as Cloudup, Droplr, Imgur, etc., to link to media.
-        Instead, attach screenshot(s) or recording(s) directly in any of the text  areas below: click, then drag and drop.
-                
+
+        Please **do not** link to image hosting services such as Cloudup, Droplr, Imgur, etc…
+        Instead, directly embed screenshot(s) or recording(s) in any of the text areas below: click, then drag and drop.
+
         If you have any questions, please ask in: `slack-bug-herders`.
-        
+
```

Comment:
Linter automatically made these whitespace changes.

I've included it in this PR because extraneous whitespace _should be_ removed.